pages/common/cvs.md

```diff
@@ -0,0 +1,32 @@
+# cvs
+
+> Concurrent Versions System, a revision control system.
+> More information: <http://cvs.nongnu.org>.
+
+- Create repository (need to manually set CVSROOT to /path/to/repo in .bashrc):
```

Comment:
As a non-CVS user, I'm really confused by this description. What is a `CVSROOT`? Why do I need to set it? And what does the configuration of a shell I don't use have to do with this? This really needs some clarification.

Reply:
@pixelcmtd 
I believe thats unclear wording on my part.
The way CVS works is that you have a CVSROOT environment variable that points to the repo path (which need not be on your local machine). The repo contains multiple projects (or modules), added via `cvs import` command.

This is different from git in the sense that you can have as many git repo on your system as you want. You switch to a different directory that has `.git` folder and further git commands will associate with that git repo. With CVS however, you define CVSROOT environment variable that points to the repo. Any cvs command irrespective of the working directory will execute with reference to path in this variable.

To be clear, the variable needs to be set _before_ any cvs command is run.

> And what does the configuration of a shell I don't use have to do with this?

I assumed the user are using bash, but yeah I get it, they could be using any shell. (I'm myself a zsh user). I suppose I could write it better as `Create repository (requires CVSROOT environment variable having path to repository)`. I'll make the changes if you say. I'm not the best at giving brief one liners.

PS.
I'm myself not a regular CVS user. Every project I need to see uses Git. I had gone through the basics of CVS some time back simply out of interest.

Reply:
Wow.

PS: I'd phrase it a bit differently like ``Create repository (requires the `CVSROOT` environment variable to be set externally)``.

Reply:
I've updated the command info :+1: